Gopkg.toml

```diff
@@ -25,3 +25,7 @@
 [[constraint]]
   name = "github.com/gogo/status"
   version = ">=v1.0.3"
+
+[[constraint]]
```

Comment:
Please add a comment explaining the need for this constraint, and the conditions under which it can be removed.